docs/about_GettingStarted.md

```diff
@@ -141,7 +141,7 @@ New-PolarisStaticRoute -RoutePath "/" -FolderPath "C:\MyAwesomeSite"
 
 Now I can just call `Start-Polaris` and the following requests will work from a browser:
 
-```
+```sh
```

Comment:
`None` is a valid option for code blocks. I usually use it for things like Output and lists like this.

Reply:
```suggestion
```None
```